### ISSUE TITLE
Issue when executing the LTE CQI Example

### ISSUE THREAD

**Issue by juansebastiani2cat** [hust-diangroup/ns3-ai#35]
Dear all,

I have installed successfully the ai module and then I tried to run the LTE CQI example but when I run it under Spyder (5.0) I get the following warning: 

I executed run_online_lstm 1 (as instructed in the example)

run_online_lstm.py:83: RuntimeWarning: Mean of empty slice.
  return (((y_pred - y_true)**2)).mean()

The returned log is empty and the MSE metrics are zero. 
![image](https://user-images.githubusercontent.com/85935421/122037918-5b64e880-cdd5-11eb-8d24-57847b89491c.png)

Additionally, when I tried running through the command line, it actually ran but the output was the following: 
![image](https://user-images.githubusercontent.com/85935421/122037710-23f63c00-cdd5-11eb-86f2-9405ff1306a2.png)

I am running it under NS3 3.30 in an Ubuntu 20.04.2 LTS. 

I appreciate your kind help. My idea is being able to use the program to obtain the metrics from NS3 and then use an AI module in Python in order to forecast traffic with it.

**Comment by juansebastiani2cat** [comment]
Hello all,

After a further experimentation with the code, I realized that the issue is that the CQI is always 15. Im guessing the NS3 simulation is not providing the proper CQI value to the python script. I hardcoded the value of the CQI to a value between 0 and 15 and instead of showing 15 like the previous example it now shows the hardcoded value that I wrote. I would truly appreciate your help with this issue. Thanks a lot.

**Comment by Mauriyin** [comment]
So currently the issue is why the CQI is always 15 during the simulation?

**Comment by juansebastiani2cat** [comment]
> So currently the issue is why the CQI is always 15 during the simulation?

Yeah, Im always obtaining 15 as CQI during the simulation, I suppose is because the simulated program does not change the CQI values for some strange reason. Could you help me get some clear light over this please ?

**Comment by PennyXQZ** [comment]
> So currently the issue is why the CQI is always 15 during the simulation?

Yes, for me the CQI is always set to be 15 as well, I guess the CQI value from ns3 might not be sent to the shared memory.
